docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -179,6 +181,159 @@ You should see something like the following:
++
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hazelcast-persistence-agent.yaml
+----
+
+. Check that Hazelcast members are ready:
++
+[source,bash]
+----
+kubectl get pods -l app.kubernetes.io/instance=hazelcast
+----
++
+[source,bash]
+----
+NAME          READY   STATUS    RESTARTS   AGE
+hazelcast-0   2/2     Running   0          2m55s
+hazelcast-1   2/2     Running   0          2m15s
+hazelcast-2   2/2     Running   0          99s
+----
+
+. Check that Hazelcast PVCs are created:
++
+[source,bash]
+----
+kubectl get pvc -l app.kubernetes.io/instance=hazelcast
+----
++
+[source,bash]
+----
+NAME                                  STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+hot-restart-persistence-hazelcast-0   Bound    pvc-116b4084-a436-4462-b413-511b77df307b   20Gi       RWO            standard       2m
+hot-restart-persistence-hazelcast-1   Bound    pvc-a7711b6b-dcbf-45cb-9577-8ce6b1892f2f   20Gi       RWO            standard       1m
+hot-restart-persistence-hazelcast-2   Bound    pvc-64314d82-da7a-4e38-bd2d-e770a63dc4e8   20Gi       RWO            standard       1m
+----
+
+=== Triggering External Backups
+
+For triggering external Backup, you need to pass two additional configurations.
+One is `bucketURI`. The other one is `secret`. According to chosen cloud provider you need to get
+bucket URI and credentials to create secret.
+
+. Create the `HotBackup` resource:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hot-backup-agent.yaml[]
+----
+<1> The bucket URI where backup data will be stored in
+<2> Name of the secret with credentials for accessing the given Bucket URI.
+
+[[create-secret]]
+. Create the secret:
++
+.. AWS: See https://docs.aws.amazon.com/sdk-for-go/api/aws/session/[AWS Session] to learn about authentication procedure.
++
+[source,bash]
+----
+kubectl create secret generic br-secret-s3 --from-literal=region=<region> \
+	--from-literal=access-key-id=<access-key-id> \
+	--from-literal=secret-access-key=<secret-access-key>
+----
+
+.. GCP: See https://cloud.google.com/docs/authentication/production/[Application Default Credentials] to learn about authentication procedure.
++
+[source,bash]
+----
+kubectl create secret generic br-secret-gcp --from-file=google-credentials-path=<service_account_json_file>
+----
+
+.. Azure: See https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal/[Azure Storage Account Keys] to learn about authentication procedure.
++
+[source,bash]
+----
+kubectl create secret generic br-secret-az \
+	--from-literal=storage-account=<storage-account> \
+	--from-literal=storage-key=<storage-key>
+----
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hot-backup-agent.yaml
+----
+
+NOTE: You can <<status-backup, check the status of a Backup >> and <<schedule-backup, schedule external Backups >> same as the local ones
```

Comment:
Maybe pull the status-backup and schedule-backup sections out of the local backup section and make them second-level headings?